exercises/rotational-cipher/src/test/java/RotationalCipherTest.java

```diff
@@ -0,0 +1,80 @@
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class RotationalCipherTest {
```

Comment:
Remember to use the [canonical data](https://github.com/exercism/x-common/blob/master/exercises/rotational-cipher/canonical-data.json) when making your tests :) That way the tests stay consistent across the different languages :)